template/src/pageComponents/mint/ContractDemo.tsx

```diff
@@ -1,16 +1,29 @@
+import { useMemo, useState } from 'react';
 import { baseSepolia } from 'viem/chains';
-import { useAccount, useContractWrite, useNetwork, usePrepareContractWrite } from 'wagmi';
+import { useNetwork } from 'wagmi';
 import { useCollectionMetadata } from '../../../onchainKit';
 import NextImage from '../../components/NextImage/NextImage';
 import ProgressBar from '../../components/ProgressBar/ProgressBar';
 import { useCustom1155Contract } from '../../hooks/contracts';
 import NotConnected from './NotConnected';
+import MintCompleteStep from './steps/MintCompleteStep/MintCompleteStep';
+import MintProcessingStep from './steps/MintProcessingStep/MintProcessingStep';
+import OutOfGasStep from './steps/OutOfGasStep/OutOfGasStep';
+import StartMintStep from './steps/StartMintStep/StartMintStep';
 import SwitchNetwork from './SwitchNetwork';
 
-const EXPECTED_CHAIN = baseSepolia;
+export const EXPECTED_CHAIN = baseSepolia;
```

Comment:
I wonder if we should have some sort of config file for this.

Or even better question, should this information come from the Hook that was generated?

Reply:
Added as constant for now. We can refactor to hook in follow up PR.